pkg/apis/controlplane/types.go

```diff
@@ -448,6 +448,43 @@ type NetworkPolicyNodeStatus struct {
 	Message string
 }
 
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// NetworkPolicyAccessReview contains the request and response for a NetworkPolicy access review.
+type NetworkPolicyAccessReview struct {
+	metav1.TypeMeta
+	Request  *NetworkPolicyAccessRequest
+	Response *NetworkPolicyAccessResponse
+}
+
+// Entity contains Namespace and Pod name as a request parameter.
+type Entity struct {
```

Comment:
If we were to extend the AccessReview in the future to support external address for example, this struct seems a bit restrictive. What about
```
type Entity struct {
     Pod *PodReference
}
```
this way in the future it could easily extend to something like
```
type Entity struct {
     Pod *PodReference
     EE  *ExternalEntityReference
}
```